acceptance-tests/dynamodb_namespace_test.go

```diff
@@ -46,50 +45,42 @@ var _ = Describe("DynamoDB Namespace", Label("dynamodb-namespace"), func() {
 
 		By("creating a table using the prefix")
 		tableName := fmt.Sprintf("csb-%s-%s", serviceInstance.GUID(), random.Hexadecimal())
-		createTablePayload := fmt.Sprintf(`{"table_name": "%s"}`, tableName)
-		response := appOne.POST(createTablePayload, "/tables")
-		Expect(response).To(HaveHTTPStatus(http.StatusAccepted))
-		defer func() {
```

Comment:
I removed this for consistency with other tests:
- when the test passes it gets cleaned up later
- if it fails, there's a cleanup script that will remove the resources

But I don't feel strongly about it and can easily add it back